netsec_fall2017/lab2/src/lab2_transport/PEEPTransport.py

```diff
@@ -8,7 +8,7 @@
 class PEEPTransport(StackingTransport):
 	logging = True
 	PEEPPacketList = []
-	logging = True
```

Comment:
I think we need set logging = True as default here right?

Reply:
because there are two same lines, I delete one